azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTest2.java

```diff
@@ -1081,17 +1083,9 @@ public void testPauseFailKill() throws Exception {
     assertThreadShutDown();
   }
 
-  private Thread runFlowRunnerInThread(final FlowRunner runner) {
+  private void runFlowRunnerInThread(final FlowRunner runner) {
     final Thread thread = new Thread(runner);
     thread.start();
-    return thread;
-  }
-
-  private void sleep(final long millisec) {
```

Comment:
Ha! I remember thinking that now I can finally remove this method, but forgot about it.